### PR TITLE
Only require one approval for minor changes

### PR DESCRIPTION
**Pull request by basil**
Waiting for two reviews for code cleanup PRs has been slowing me down recently. Such PRs with no user-visible changes don't tend to attract very many reviewers, with or without explicit solicitation, nor do the PRs tend to change very much after the second review. Seems to me that we can dispense with the heavyweight process for code cleanup PRs while still maintaining the 24-hour waiting period to allow for objections to be raised.

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Review by StefanSpieker — APPROVED** [review]

**Review by jglick — APPROVED** [review]

**Comment by amuniz** [comment]
Seems a bit risky to me. Sometimes minor changes lead to huge problems. Also, what's the definition of minor?
No strong opinion, just thinking out loud.

**Comment by timja** [comment]
> Seems a bit risky to me. Sometimes minor changes lead to huge problems. Also, what's the definition of minor? No strong opinion, just thinking out loud.

No offence but you haven't been reviewing the changes. If there were more reviewers active then this wouldn't be a PR.

**Comment by amuniz** [comment]
Fair enough.

**Review by oleg-nenashev — APPROVED** [review]
I am +1 for this change. I would suggest updating the wording to indicate that we actually target "small changes" as noted by @basil in the pull request title. `skip-changelog` is IMHO not enough, because sometimes we have big and sometimes risky refactorings with this label. I'd like to also add small non-functional changes (like typo fixes) to the scope of single-approval

P.S: At the moment I cannot dedicate enough time to core PR reviews due to personal reasons. The Jenkins governance board members and @timja know the details

File: docs/MAINTAINERS.adoc
Comment (timja) [inline docs/MAINTAINERS.adoc:377]:
```suggestion
* Only one approval is required for low-risk small changes with the `skip-changelog` label,
```

Comment (basil) [reply docs/MAINTAINERS.adoc]: LGTM

File: docs/MAINTAINERS.adoc
Comment (daniel-beck) [inline docs/MAINTAINERS.adoc:378]:
```suggestion
  as long as both author and approver have write access to the repository.
```

?

Comment (basil) [reply docs/MAINTAINERS.adoc:378]: I'm fine with tightening this constraint, since in practice I haven't seen any changes piling up from outside contributors. The one exception I can think of is minor Dependabot changes (e.g. updates to test dependencies) which I have seen pile up a few times. Ideally I would like to invoke this exception in cases like that. Technically the robot has write access to the repository, but to clarify your intent: would that seem alright to you?

Comment (daniel-beck) [reply docs/MAINTAINERS.adoc:378]: Probably depends on the bump. Assuming we have dependabot only doing trivial stuff and exclude the rest, that would be fine.

Comment (basil) [reply docs/MAINTAINERS.adoc:378]: Yeah, there's inherently some degree of subjectivity in things like this. Sometimes even two reviews and 24 hours are not enough (e.g., for Groovy version bumps). But given the number of active reviewers and the fact that many of the original authors of the code have moved on, I think this new rule strikes a reasonable compromise.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by daniel-beck — APPROVED** [review]

**Pull request merged**
